src/main/kotlin/gdx/liftoff/data/platforms/Lwjgl3.kt

```diff
@@ -144,25 +144,24 @@ class Lwjgl3GradleFile(val project: Project) : GradleFile(Lwjgl3.ID) {
     addDependency("com.badlogicgames.gdx:gdx-platform:\$gdxVersion:natives-desktop")
   }
 
-  override fun getContent(): String = """buildscript {
+  // language=groovy
```

Comment:
[Language injection](https://www.jetbrains.com/help/idea/using-language-injections.html) for IDEA, will highlight the string as if it was groovy:

![image](https://github.com/libgdx/gdx-liftoff/assets/1263058/e580730e-f02a-4b65-a1ab-aca04704d992)

Reply:
I had no idea this was possible! There are so many places I could use this. Thanks!